# This message contains the thrust f_t and the three torques tau_x, tau_y, and tau_z

Header header

float64 f_t
float64 tau_x
float64 tau_y
float64 tau_z
